doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -171,10 +171,12 @@ bash get_source.sh
 :penguin:
 When you have all the source files that you need, run the configure script, which detects how to build in the current build environment.
 ```
-bash configure --with-freemarker-jar=/<my_home_dir>/freemarker.jar
+bash configure --with-freemarker-jar=/<my_home_dir>/freemarker.jar --with-boot-jdk=/usr/lib/adoptojdk-java-11
 ```
 :warning: You must give an absolute path to freemarker.jar
 
+:warning: The path in the example --with-boot-jdk= option is appropriate for the Docker installation, this option is not required if you followed the instructions to download and setup the boot JDK.
```

Comment:
I would reword this: the option is required if `JAVA_HOME` isn't set or doesn't refer to a suitable boot jdk. If you're building multiple jdk versions, you probably can't satisfy them all with `JAVA_HOME`. I think we should advise use of `--with-boot-jdk` in general.

Reply:
I've updated, but if you still want changes we can figure it out on Slack.